exercises/change/test/Tests.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Change (findFewestCoins)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "Change" $
+          describe "change" $ for_ cases test
+  where
+
+    test Case{..} = it description assertion
+      where
+        assertion  = expression `shouldBe` expected
+        expression = findFewestCoins target coins
+
+-- Test cases adapted from `exercism/x-common/change/canonical-data.json` on 2016-09-17.
```

Comment:
Just a note!

The address is now `exercism/x-common/exercises/canonical-data.json`, which is ridiculously long, so I usually just write:
```
-- Test cases adapted from `exercism/x-common` on 2016-09-17.
```